modules/composer-service-api/src/main/java/org/opencastproject/composer/api/ComposerService.java

```diff
@@ -258,4 +260,28 @@ Job image(Track sourceTrack, String profileId, Map<String, String> properties) t
    */
   Job demux(Track sourceTrack, String profileId) throws EncoderException, MediaPackageException;
 
+  /**
+   * Reads a smil definition and create one media track in multiple delivery formats. The track in the smil is selected
+   * by "trackParamGroupId" which is the paramGroup in the smil The multiple delivery formats are determined by a list
+   * of encoding profiles by name. The resultant tracks will be tagged by profile name. The smil file can contain more
+   * than one source track but they must have the same dimension. This is used mainly on smil.xml from the editor. There
```

Comment:
Remove "There is a 2s fadein/fadeout between each clip" or replace with correct comment (the transitionDuration is configurable)